### PR TITLE
Revise "sentiment-analysis-cnn_vn" - Phần 2

### LINKED ISSUE #2767: Revise "sentiment-analysis-cnn_vn" - Phần 2
Phần này được dịch bởi:
* **lyphilong**
* **phattruongdevpanel**
* **longnguyen1984**

Nếu bạn đã dịch phần này, vui lòng bỏ qua việc revise.

### PR DESCRIPTION
**Pull request by quangnhat185**
Close #2767

### CODE REVIEW

File: chapter_natural-language-processing-applications/sentiment-analysis-cnn_vn.md [hunk 1]
```diff
@@ -232,7 +232,7 @@ it usually allows the model to be unaffected by the manually added characters.
 
 Để cải thiện chất lượng tính toán, ta thường kết hợp những mẫu thời gian có độ dài khác nhau vào một minibatch và làm cho chiều dài của từng mẫu thời gian đồng nhất bằng cách thêm các ký tự đặc biệt (ví dụ 0) vào cuối những mẫu ngắn hơn.
 Đương nhiên, các ký tự được thêm vào không làm thay đổi bản chất ngữ nghĩa.
-Bởi vì, mục tiêu chính của tầng gộp cực đại theo thời gian là học được những đặc trưng quan trọng của thời gian, điều đó thường cho phép mô hình không bị ảnh hưởng bởi các ký tự được thêm vào thủ công.
+Bởi vì, mục tiêu chính của tầng gộp cực đại theo thời gian là học được những đặc trưng quan trọng của thời gian, điều đó thường cho phép mô hình không bị ảnh hưởng bởi các ký tự đư\speợc thêm vào thủ công.
```
Comment (quangnhat185) [inline chapter_natural-language-processing-applications/sentiment-analysis-cnn_vn.md:235]:
```suggestion
Bởi vì, mục tiêu chính của tầng gộp cực đại theo thời gian là học được những đặc trưng quan trọng của thời gian, điều đó thường cho phép mô hình không bị ảnh hưởng bởi các ký tự được thêm vào thủ công.
```

File: chapter_natural-language-processing-applications/sentiment-analysis-cnn_vn.md [hunk 2]
```diff
@@ -464,10 +464,10 @@ tuning hyperparameters, using larger pre-trained word vectors, and using the spa
 3. What other natural language processing tasks can you use textCNN for?
 -->
 
-1. Điều chỉnh các tham số mô hình và so sánh hai phương pháp phân tích hai cảm xúc, sử dụng các mạng nơ-ron truy hồi và các mạng nơ-ron tích chập, ở khía cạnh độ chính xác và hiệu suất tính toán.
+1. Điều chỉnh các tham số mô hình và so sánh phương pháp phân tích hai cảm xúc giữa mạng nơ-ron truy hồi và mạng nơ-ron tích chập, xét trên khía cạnh độ chính xác và hiệu suất tính toán.
```
Comment (quangnhat185) [inline chapter_natural-language-processing-applications/sentiment-analysis-cnn_vn.md:467]:
```suggestion
1. Điều chỉnh các tham số mô hình và so sánh phương pháp phân tích hai cảm xúc giữa mạng nơ-ron truy hồi và mạng nơ-ron tích chập xét trên khía cạnh độ chính xác và hiệu suất tính toán. 
```

File: chapter_natural-language-processing-applications/sentiment-analysis-cnn_vn.md [hunk 3]
```diff
@@ -295,16 +295,16 @@ Finally, we use a fully connected layer to transform the 9-dimensional vector
...
 
-:numref:`fig_conv1d_textcnn` minh họa một ví dụ cho textCNN.
-Đầu vào ở đây là một câu gồm 11 từ, với mỗi từ được biểu diễn bằng một vector từ 6 chiều.
-Vì vậy, câu đầu vào có độ rộng là 11 và số kênh đầu vào là 6
-Chúng ta giả sử rằng có 2 hạt nhân của tích chập một chiều có độ rộng là 2 và 4, tương ứng với số kênh đầu ra là 4 và 5.
+:numref:`fig_conv1d_textcnn` minh họa một ví dụ cho textCNN. 
+Đầu vào ở đây là một câu gồm 11 từ, với mỗi từ được biểu diễn bằng một vector từ 6 chiều. 
+Vì vậy, câu đầu vào có độ rộng là 11 và số kênh đầu vào là 6.
+Chúng ta giả sử rằng 2 hạt nhân tích chập một chiều có độ rộng lần lượt là 2 và 4, tương ứng với số kênh đầu ra là 4 và 5. 
 Cho nên sau phép tính tích chập một chiều, độ rộng của đầu ra có 4 kênh là $11-2+1=10$,
```
Comment (rootonchair) [inline chapter_natural-language-processing-applications/sentiment-analysis-cnn_vn.md:302]:
```suggestion
Cho nên sau phép tính tích chập một chiều, đầu ra 4 kênh có chiều rộng là là $11-2+1=10$,
```

File: chapter_natural-language-processing-applications/sentiment-analysis-cnn_vn.md [hunk 4]
```diff
@@ -295,16 +295,16 @@ Finally, we use a fully connected layer to transform the 9-dimensional vector
...
 trong khi đó độ rộng của đầu ra còn lại 5 kênh là $11-4+1=8$.
```
Comment (rootonchair) [inline chapter_natural-language-processing-applications/sentiment-analysis-cnn_vn.md:303]:
```suggestion
trong khi đó độ rộng của đầu ra 5 kênh còn lại là $11-4+1=8$.
```

File: chapter_natural-language-processing-applications/sentiment-analysis-cnn_vn.md (hunk 2 above)
Comment (rootonchair) [inline chapter_natural-language-processing-applications/sentiment-analysis-cnn_vn.md:467]:
```suggestion
1. Điều chỉnh các tham số mô hình và so sánh hai phương pháp phân tích cảm xúc giữa mạng nơ-ron truy hồi và mạng nơ-ron tích chập, xét trên khía cạnh độ chính xác và hiệu suất tính toán. 
```

**Review by rootonchair — COMMENTED** [review]
Thanks anh

File: chapter_natural-language-processing-applications/sentiment-analysis-cnn_vn.md [hunk 5]
```diff
@@ -501,3 +501,4 @@ Tên đầy đủ của các reviewer có thể được tìm thấy tại https
 
 <!-- Phần 5 -->
 * Nguyễn Mai Hoàng Long
+* Nguyễn Lê Quang Nhật
```
Comment (quangnhat185) [inline chapter_natural-language-processing-applications/sentiment-analysis-cnn_vn.md:504]:
```suggestion
* Nguyễn Lê Quang Nhật
* Phạm Hồng Vinh
```

File: chapter_natural-language-processing-applications/sentiment-analysis-cnn_vn.md [hunk 6]
```diff
@@ -265,11 +265,11 @@ Convolution kernels with different widths may capture the correlation of differe
 A dropout layer can be used in this step to deal with overfitting.
 -->
 
-1. Định nghĩa nhiều hạt nhân một chiều nhằm dùng để thực hiện các phép tính tích chập trên đầu vào.
-Những hạt nhân tích chập với độ rộng khác nhau có thể học được sự tương quan của số lượng từ liền kề khác nhau.
-2. Thực hiện gộp cực đại theo thời gian trên tất cả các kênh đầu ra, sau đó nối các giá trị gộp được của các kênh này thành một vector
-3. Vector vừa nối sẽ được biến đổi thành đầu ra cho từng hạng mục bằng các đưa qua tầng kết nối đầy đủ.
-Có thể sử dụng tầng dropout ở bước này để giải quyết tình trạng quá khớp.
+1. Định nghĩa nhiều hạt nhân tích chập một chiều để thực hiện các phép tính tích chập trên đầu vào. 
+Những hạt nhân tích chập với độ rộng khác nhau có thể học được sự tương quan của số lượng từ liền kề khác nhau.
```
Comment (lkhphuc) [inline chapter_natural-language-processing-applications/sentiment-analysis-cnn_vn.md:269]:
```suggestion
Những hạt nhân tích chập với độ rộng khác nhau có thể học được sự tương quan của các cụm từ liền kề với số lượng khác nhau. 
```

File: chapter_natural-language-processing-applications/sentiment-analysis-cnn_vn.md [hunk 7]
```diff
@@ -265,11 +265,11 @@ Convolution kernels with different widths may capture the correlation of differe
...
 
-1. Định nghĩa nhiều hạt nhân một chiều nhằm dùng để thực hiện các phép tính tích chập trên đầu vào.
-Những hạt nhân tích chập với độ rộng khác nhau có thể học được sự tương quan của số lượng từ liền kề khác nhau.
-2. Thực hiện gộp cực đại theo thời gian trên tất cả các kênh đầu ra, sau đó nối các giá trị gộp được của các kênh này thành một vector
-3. Vector vừa nối sẽ được biến đổi thành đầu ra cho từng hạng mục bằng các đưa qua tầng kết nối đầy đủ.
-Có thể sử dụng tầng dropout ở bước này để giải quyết tình trạng quá khớp.
+1. Định nghĩa nhiều hạt nhân tích chập một chiều để thực hiện các phép tính tích chập trên đầu vào. 
+Những hạt nhân tích chập với độ rộng khác nhau có thể học được sự tương quan của số lượng từ liền kề khác nhau. 
+2. Thực hiện gộp cực đại theo thời gian trên tất cả các kênh đầu ra, sau đó nối các giá trị gộp được của các kênh này thành một vector. 
+3. Vector nối trên sẽ được biến đổi thành đầu ra cho từng hạng mục bằng các đưa qua tầng kết nối đầy đủ.
```
Comment (lkhphuc) [inline chapter_natural-language-processing-applications/sentiment-analysis-cnn_vn.md:271]:
```suggestion
3. Vector nối trên sẽ được biến đổi thành đầu ra cho từng hạng mục bằng thông qua tầng kết nối đầy đủ. 
```

File: chapter_natural-language-processing-applications/sentiment-analysis-cnn_vn.md [hunk 8]
```diff
@@ -464,10 +464,10 @@ tuning hyperparameters, using larger pre-trained word vectors, and using the spa
 3. What other natural language processing tasks can you use textCNN for?
 -->
 
-1. Điều chỉnh các tham số mô hình và so sánh hai phương pháp phân tích hai cảm xúc, sử dụng các mạng nơ-ron truy hồi và các mạng nơ-ron tích chập, ở khía cạnh độ chính xác và hiệu suất tính toán.
+1. Điều chỉnh các tham số mô hình và so sánh hai phương pháp phân tích cảm xúc giữa mạng nơ-ron truy hồi và mạng nơ-ron tích chập, xét trên khía cạnh độ chính xác và hiệu suất tính toán. 
 2. Bạn có thể cải thiện thêm độ chính xác của mô hình trên tập kiểm tra thông qua việc sử dụng ba phương pháp đã được giới thiệu ở phần trước:
-điều chỉnh các tham số mô hình, sử dụng các vector từ tiền huấn luyện lớn hơn, và sử dụng công cụ token hóa từ spaCy.
-3. Những tác vụ xử lý ngôn ngữ tự nhiên khác nào mà bạn có thể sử dụng textCNN cho chúng?
+điều chỉnh các tham số mô hình, sử dụng các vector từ tiền huấn luyện lớn hơn, và sử dụng công cụ token hóa từ spaCy. 
+3. Những tác vụ xử lý ngôn ngữ tự nhiên khác nào mà bạn có thể sử dụng textCNN cho chúng?
```
Comment (lkhphuc) [inline chapter_natural-language-processing-applications/sentiment-analysis-cnn_vn.md:470]:
```suggestion
3. Bạn còn có thể sử dụng TextCNN cho những tác vụ xử lý ngôn ngữ tự nhiên nào khác? 
```

**Review by lkhphuc — APPROVED** [review]

File: chapter_natural-language-processing-applications/sentiment-analysis-cnn_vn.md [hunk 9]
```diff
@@ -501,3 +501,5 @@ Tên đầy đủ của các reviewer có thể được tìm thấy tại https
 
 <!-- Phần 5 -->
 * Nguyễn Mai Hoàng Long
+* Nguyễn Lê Quang Nhật
+* Phạm Hồng Vinh
```
Comment (quangnhat185) [inline chapter_natural-language-processing-applications/sentiment-analysis-cnn_vn.md:505]:
```suggestion
* Phạm Hồng Vinh
* Lê Khắc Hồng Phúc
```

**Review by duythanhvn — APPROVED** [review]

**Pull request merged**